kustomize/aks/dashboard.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: racetrack
+  name: dashboard
+  labels:
+    app.kubernetes.io/name: dashboard
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: dashboard
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: dashboard
+    spec:
+      securityContext:
+        fsGroup: 200000
+        runAsUser: 100000
+        runAsGroup: 100000
+      hostname: dashboard
+      subdomain: dashboard
+      priorityClassName: high-priority
+      containers:
+        - name: dashboard
+          image: ghcr.io/theracetrack/racetrack/dashboard:latest
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 7003
+          tty: true
+          envFrom:
+            - configMapRef:
+                name: postgres-env
+            - configMapRef:
+                name: lifecycle-env
+          env:
+            - name: LIFECYCLE_URL
+              value: 'http://lifecycle:7002/lifecycle'
+            - name: EXTERNAL_LIFECYCLE_URL
+              value: 'http://1.1.1.1/lifecycle'
+            - name: EXTERNAL_PUB_URL
+              value: 'http://1.1.1.1/pub'
+            - name: DJANGO_DEBUG
+              value: 'true'
+            - name: AUTH_REQUIRED
+              value: 'true'
+            - name: DASHBOARD_PORT
+              value: '7003'
+            - name: LIFECYCLE_AUTH_TOKEN
+              value: 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzZWVkIjoiMjIzYThhM2ItNzBkOC00YzA4LWI4YmYtNzlkNDA3MDI3ZGE2Iiwic3ViamVjdCI6ImRhc2hib2FyZCIsInN1YmplY3RfdHlwZSI6ImludGVybmFsIiwic2NvcGVzIjpbImZ1bGxfYWNjZXNzIl19.CAww41hboBPHNflJxKBNRnO-Eo9Y_EKsfU66mK3kdk4'
```

Comment:
It seems like a mistake to have a token stored like this. Even if it's just the default value, perhaps that'd be okay to ask the users to fill out for themselves.

Reply:
Actually, this token is not a user's token, but the one for internal communication of components and it's more complicated to generate this one. Plus, notice that each of them is different. There's 3 of them: one for each dashboard, image-builder and pub.
They can be generated offline, before launching the Racetrack, but the process is a bit complicated. That's why I omitted it and went with the default ones.
Do you think we should put that as well in this documentation?

Reply:
I think it's fine from a documentation standpoint then. Double checking, we do in fact recommend changing these for deploying to production, right? Because otherwise they wouldn't really be any good for authenticating anyone.

Reply:
I'm not crazy about having a token here either, but I can see how automating it away might be a whole heap of new complexity.

What a project like Django does with its `SECRET_KEY` is, it autogenerates it.

Maybe a short-term low-tech option here is to leave it blank and instruct the user to fill it out during deployment?

Reply:
Okay, I'm gonna add a section about how to generate these keys and where to fill it out. 

Django generates `SECRET_KEY` when creating a new project. We don't have that step and we have to mention that step in the docs.

Reply:
I included steps for generating secrets and tokens.